OracleTuxedo/ART_SALT_TMA/samples/README.md

```diff
@@ -0,0 +1,33 @@
+
+TuxedoART TMA samples on Docker
+===============
+
+## Configuration 
+## for TMA SNA
+1. Configure `sna.in` file in tma_samples.zip if neccessary.
+wasa                                   #Mainframe Host Name
+12200                                  #Mainframe Port Number
+VT210                                  #Stack Type
+CRMLU09                                #Local LU
+CICSA                                  #Remote LU
+CR09                                   #Local SYSID
+CICS                                   #Remote SYSID
+CRM12200                               #SNA Group
+/home/oracle/tuxHome/tuxedo12.2.2.0.0  #Tuxedo Path
+2. Make sure that CRM on z/OS is configured and running properly, which could be referred to edoc.
+## for TMA TCP
+1. Configure `tcp.in` file in tma_samples.zip if neccessary.
+wasa                                   #Mainframe Host Name
+/home/oracle/tuxHome/tuxedo12.2.2.0.0  #Tuxedo Path
+2. Make sure that gateway for CICS and IMS on z/OS are configured and running properly, which could be referred to edoc.
+
+## How to build
+build.sh
+
+## How to run
+You can start the image in a new container with:
+docker run --privileged -d -p 11122:22 -p 18080:8080 -e DISPLAY=bej301699.cn.oracle.com:1.0 -v /tmp/.X11-unix:/tmp/.X11-unix -h arthost --name tuxedoartrttmasample oracle/tuxedoartrttmasample:12.2.2 /sbin/init
```

Comment:
As above, we cannot accept any Dockerfile that requires `--privileged` access. Nor should you be running `/sbin/init`. This is a container, not a VM.